pages/linux/gdrive.md

```diff
@@ -0,0 +1,19 @@
+# gdrive
+
+> Terminal tool for Google Drive operations. Folder/file id can be obtained from the Google Drive folder or id url.
+
+- Upload local 'path' to -p 'id' parent folder:
```

Comment:
Not sure if `-p` is really required here. Or the quote marks around `path` and `id`. Perhaps something like this? (If this is what it does)

> Upload a local path to the parent folder with the specified id: